Generator/Assets/Items.cs

```diff
@@ -1157,6 +1157,24 @@ private int SetupItemPoolPoes(SharedSettings parseSetting)
                     startingPoes += 1;
             }
 
+            int plandoPoes = 0;
+            foreach ((string checkName, Item item) in parseSetting.plandoChecks)
+            {
+                switch (item)
+                {
```

Comment:
This just needs to be an if-statement instead of a switch-case. Looks good other than that.